src/mvt_sql.py

```diff
@@ -182,32 +182,6 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec, extent):
     groupby = ",".join(['"%s"' % name for name in column_names])
 
     if vec == "polygon":
-        coastline_query = """select ST_AsMVTGeom(geom, ST_TileEnvelope(%s, %s, %s), %s, 64, true) as way, %s from
```

Comment:
ok, I don't think merging coastline query into generic polygon query is the right move since coastline is stored as subdivided geometry and always need to be union.